File: Data Structures & Algorithms/two-integer-sum/submission-0.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        length = len(nums)
        for i in range(0, length):
            for j in range(i + 1, length):
                l1 = nums[i]
                l2 = nums[j]
                if (target - (l1 + l2)) == 0:
                    return [i, j]
                

File: Data Structures & Algorithms/concatenation-of-array/submission-0.rs
impl Solution {
    pub fn get_concatenation(nums: Vec<i32>) -> Vec<i32> {
    let mut new_arr = vec![0; nums.len() * 2];
    for i in 0..new_arr.len() {
        new_arr[i] = nums[i % nums.len()];
    }
    new_arr
    }
}
